Start writing a better README (#144)
elixir-europe/infectious-diseases-toolkit: README.md

## Contribute

IDTk is an open [community](https://www.infectious-diseases-toolkit.org/about/contributors) project, and you are welcome to join us! The content of the material is developed in Markdown and a templating system ([Jekyll](http://jekyllrb.com/)) is used to format the Markdown pages and generate a website at ([https://www.infectious-diseases-toolkit.org](https://www.infectious-diseases-toolkit.org)).

Do you want to help with this project? Please check out following pages for more information:

* [Code of Conduct](https://www.infectious-diseases-toolkit.org/CODE_OF_CONDUCT)
* [How to contribute](https://www.infectious-diseases-toolkit.org/contribute/)
* [Deploy the website locally](https://www.infectious-diseases-toolkit.org/contribute/working-with-git#the-advantage-of-working-locally-previewing-your-changes-through-your-web-browser)

## How to reach us

> Do you have any question, a content proposal or see a mistake?

Author: hmmm I wonder if it also covers 'I just want to file a problem', like: This link is not working.

* Feel free to open a [GitHub issue](https://github.com/elixir-europe/infectious-diseases-toolkit/issues)
* Or contact the editors of this project through [idtk-editors@elixir-europe.org](mailto:rdm-editors@elixir-europe.org )

## IDTk is a BY-COVID project

The BeYond-COVID project aims to make COVID-19 data accessible to scientists in laboratories but also to anyone who can use it, such as medical staff in hospitals or government officials.

Going beyond SARS-CoV-2 data, the project will provide a framework for making data from other infectious diseases open and accessible to everyone.

## License

The process documents and data are made available under a CC-BY license. Software are made available under an MIT license. More information about our license can be found on our [license](LICENSE) page.

## How to cite the IDTk
We have not published the story of IDTk yet. Till then, please refer to:

<div style="padding:30px;"><p><i>Infectious Diseases Toolkit. A deliverable from the EU-funded BY-COVID project (grant agreement number 101046203). URL: <a href="https://www.infectious-diseases-toolkit.org/">https://www.infectious-diseases-toolkit.org/</a>.</i></p>
</div>

> The Infectious Diseases Toolkit makes all of their materials publicly available under Open Source Initiative licenses. The process documents and data are made available under a CC-BY license. Software are made available under an MIT license. For full details on licensing, please visit our License document.

Reviewer: ```suggestion
All of the material from the IDTk is made publicly available under Open Source Initiative licences. The process documents and data are made available under a CC-BY license. Software is made available under an MIT licence. For full details on licensing, please see our [licence document](link to licence).
```

## Acknowledgements

> The Infectious Diseases Toolkit is developed in BY-COVID that received funding from the European Union’s Horizon Europe Research and Innovation Programme under grant agreement number 101046203.It is built based on the experience and ways of working of the [RDMkit](https://rdmkit.elixir-europe.org/) and also makes use of the [ELIXIR Toolkit Theme](https://elixir-belgium.github.io/elixir-toolkit-theme/) for deployment.

Reviewer: ```suggestion
The IDTk is developed as part of the project BY-COVID, which received funding from the European Union’s Horizon Europe Research and Innovation Programme (grant number: 101046203). It is built based on experience and ways of working from the [RDMkit](https://rdmkit.elixir-europe.org/), and makes use of the [ELIXIR Toolkit Theme](https://elixir-belgium.github.io/elixir-toolkit-theme/) for deployment.
```
